app/views/content/landing/campus-advisers/_adviser.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="row">
   <div class="grey col-space-l">
     <section class="col col-720 col-space-0">
-         <%= render Content::AdviserComponent.new(
+         <%= render Content::.new(
```

Comment:
Think this has been included by mistake - don't think you should be updating this landing page as part of this PR?

The change is causing build errors as well - if you can back it out, your build errors should disappear :-)